pages/[type]/[id].vue

```diff
@@ -290,10 +290,9 @@
             </p>
           </div>
           <div
-            v-highlightjs
             class="markdown-body"
             v-html="
-              $xss(
+              renderString(
                 $md.render(
                   project.moderator_message.body
                     ? project.moderator_message.body
```

Comment:
Look at /index.vue (in this directory) for an example on this func.

It should just be renderString(project.moderator_message.body ? ... (whatever else). Also make sure to import the function (you can also see index.vue for the file to import from)